units/crons_remote_hour.go

```diff
@@ -57,6 +57,7 @@ func (j *cronsRemoteHourJob) Run(ctx context.Context) {
 		PopulateSSHKeyUpdates(j.env),
 		PopulateDuplicateTaskCheckJobs(),
 		PopulatePodResourceCleanupJobs(),
+		PopulateUnexpirableSpawnHostStatsJob(),
```

Comment:
It doesn't really seem like this needs to run more than daily (similar to [DEVPROD-733](https://jira.mongodb.org/browse/DEVPROD-733) i think a bunch of these really don't need to run so much). If it's not too much trouble I'd make a remote_daily, or at least comment this on DEVPROD-733 to be moved, unless we really need it hourly.

Reply:
Being in the hourly cron doesn't necessarily mean that they but they _run_ every hour but that they try to enqueue a job every hour. The timestamp that they use for rounding the job ID determines how often the job actually runs because it produces a conflict on the job `_id` if it's the same time period. If the interval only enqueued once per day (which is what a daily job would do), it may produce some issues because if you _need_ it to run every day, a daily job will try to enqueue at most once per day, so if it fails once, you're done and there's no hope to run it that day.

We could tune it to be once every few hours, but honestly, I think it's fine as-is. The timestamp rounding already ensures that the jobs actually run at the correct cadence, and trying to enqueue a few extra jobs per hour (that won't run) produces negligibly low on the DB.